src/static_ir/static_ir.jl

```diff
@@ -52,6 +52,7 @@ function generate_generative_function(ir::StaticIR, name::Symbol, options::Stati
             params::Dict{Symbol,Any}
         end
         (gen_fn::$gen_fn_type_name)(args...) = $(GlobalRef(Gen, :propose))(gen_fn, args)[3]
+        $(GlobalRef(Gen, :get_ir))(::$gen_fn_type_name) = $(QuoteNode(ir))
```

Comment:
This seems reasonable, but it's making me wonder whether there was any reason we didn't have this in the first place. I guess the answer is probably that `get_ir` was mostly used from `@generated` functions, which only had access to the type of the GF, not the GF itself.